frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/DataConnectionFolderPathField.tsx

```diff
@@ -11,9 +11,9 @@ const DataConnectionFolderPathField: React.FC<DataConnectionFolderPathFieldProps
   folderPath,
   setFolderPath,
 }) => {
-  type validate = React.ComponentProps<typeof FormGroup>['validated'];
+  type Validate = React.ComponentProps<typeof FormGroup>['validated'];
```

Comment:
In case someone missed this in the PR description - changing a type variable to PascalCase that I missed in previous PR-https://github.com/opendatahub-io/odh-dashboard/pull/1512#pullrequestreview-1549935352